prac2-pages/Simranpreet Kaur/login_page.dart

```diff
@@ -0,0 +1,181 @@
+// ignore_for_file: prefer_const_literals_to_create_immutables, prefer_const_constructors
+
+import 'package:flutter/material.dart';
+import 'login2.dart' ;
+
+class LoginPage extends StatefulWidget{
+  const LoginPage({Key? key}) : super(key:key);
+
+  @override
+  State<LoginPage> createState() => _LoginPageState();
+
+}
+class _LoginPageState extends State<LoginPage> {
+  bool check1 = true;
+  @override
+  Widget build(BuildContext context){
+    return Scaffold(
+      backgroundColor: Colors.white,
+      body: SafeArea(
+        child : Center(
+            child: Column(
+              children: [
+            SizedBox(height:55),
+             //Login
+             Text(
+               'Login',
+             style: TextStyle(
+               fontWeight: FontWeight.bold,
+               fontSize: 30,
+               color: Color(0xFF06061C),
+             ),
+             ),
+            SizedBox(height: 25),
+        //Create an account : Signup
+            Text('Create an account',
+              style:TextStyle(
+                color: Colors.deepPurpleAccent,
+                fontWeight: FontWeight.normal,
+                fontSize: 20,
+              ),
+            ),
+            SizedBox(height: 30,),
+            Container(
```

Comment:
Try to avoid using containers unnecessarily, as they are resource heavy and end up decreasing the performance of the app.
- You may refer to the docs for such information.

Reply:
Resolved! :+1: